src/context/yaml/handlers/brandingTemplates.js

```diff
@@ -0,0 +1,57 @@
+import fs from 'fs-extra';
+import path from 'path';
+import { constants, loadFileAndReplaceKeywords } from '../../../tools';
+
+async function parse(context) {
+  // Load the HTML file for each page
+
+  const { branding } = context.assets;
+
+  if (!branding || !branding.templates) return { branding: { ...branding, templates: undefined } };
+
+  const templates = branding.templates.map((templateDefinition) => {
+    const markupFile = path.join(templateDefinition.body);
+    return {
+      template: templateDefinition.template,
+      body: loadFileAndReplaceKeywords(markupFile, context.mappings)
+    };
+  });
+
+  return {
+    branding: {
+      ...branding,
+      templates
+    }
+  };
+}
+
+async function dump(context) {
+  const { branding } = context.assets || { branding: undefined };
```

Comment:
Similarly to my point above, it looks like there's a shared concerns about the `branding` property between this handler and the existing branding handler. I think it would be sensical and more clear to combine together.

Reply:
:100: